package.json

```diff
@@ -33,13 +33,21 @@
     "lint": "eslint .",
     "flow": "flow",
     "fix": "eslint . --fix",
-    "build:docs": "markdox lib/plugin-api/hydrogen-provider.js lib/plugin-api/hydrogen-kernel.js -o PLUGIN_API.md"
+    "prettier": "prettier --single-quote --write 'lib/**/*.js' 'spec/**/*.js'",
+    "build:docs": "markdox lib/plugin-api/hydrogen-provider.js lib/plugin-api/hydrogen-kernel.js -o PLUGIN_API.md",
+    "precommit": "lint-staged"
   },
   "repository": "https://github.com/nteract/hydrogen",
   "license": "MIT",
   "engines": {
     "atom": ">=1.6.0 <2.0.0"
   },
+  "lint-staged": {
+    "*.js": [
+      "prettier --single-quote --write",
```

Comment:
I used single quotes here to keep the diff smaller, though I'm happy to change that.